src/vmm/src/device_manager/persist.rs

```diff
@@ -119,6 +134,23 @@ impl<'a> Persist<'a> for MMIODeviceManager {
                 continue;
             }
 
+            #[cfg(target_arch = "aarch64")]
+            {
+                if *device_id == DeviceType::Serial.to_string()
+                    || *device_id == DeviceType::RTC.to_string()
```

Comment:
nit: you can match on `device_type` directly instead of `device_id` and string transformations